license_copyright/license.yml

```diff
@@ -27,11 +27,12 @@ runs:
       run: |
         git checkout -b license-update-${{ github.sha }}
         git commit -m "docs: updating copyright license year"
-        git push
+        git push origin license-update-${{ github.sha }}:license-update-${{ github.sha }} || true
```

Comment:
I think you shouldn't use `|| true` because it can fail silently and no one would notice